client/src/main/java/io/pravega/client/batch/BatchClient.java

```diff
@@ -32,12 +30,16 @@
 public interface BatchClient {
 
     /**
-     * Provides a list of segments and their metadata for a given stream.
+     * Provide a list of segments for a given stream between fromStreamCut and toStreamCut.
+     * Passing null to fromStreamCut and toStreamCut will result in using the start of stream and the current end of
```

Comment:
Please explain precisely how the start/end of stream is obtained. Also, there is no mention of what happens in the case I use stream cuts that touch truncated data.

Reply:
done.